### ISSUE TITLE
Not compatible with Python3

### ISSUE THREAD

**Issue by OscarGibson** [pmaigutyak/mp-liqpay#1]
After installing and adding 'liqpay' to "INSTALLED_APPS" I have the next error:

> from urlparse import urljoin
> ModuleNotFoundError: No module named 'urlparse'

That happened because I have Python version 3, and here we must import **urljoin** from **urllib.parse**, not from **urlparse**

I believe that the simple check would be the best idea:

In `liqpay/__init__.py` file instead this 
```
urlparse import urljoin
```
add this code 
```
try:
    ## For Python2.x
    from urlparse import urljoin
except Exception as e:
    ## For Python3.x
    from urllib.parse import urljoin
```

**Comment by pmaigutyak** [comment]
Hi, I've made fix and pushed it to version 1.1.

**Comment by pmaigutyak** [comment]
Let me know is problem resolved
